src/backend/gl/src/queue.rs

```diff
@@ -313,11 +322,12 @@ impl CommandQueue {
             };
         } else if self.state.num_viewports > 1 {
             // 16 viewports is a common limit set in drivers.
-            let viewports: SmallVec<[[f32; 4]; 16]> = (0..self.state.num_viewports)
+            let viewports: SmallVec<[[f32; 4]; 16]> = (0 .. self.state.num_viewports)
```

Comment:
why are we changing this back? I like the spaces, but rustfmt stable doesn't :/

Reply:
I was just using nightly and rustfmt so that it matched what was in the `rustfmt.toml`. We could use stable or we could take out the nightly-only formatting options from the toml so that they both yield the same formatting ( which would probably be best ).

Reply:
we should take out the nightly options, yeah (unfortunately)

Reply:
I'll ad that to this PR then.

Reply:
Ideally, to a different PR, if possible.
This one just doesn't need to touch the formatting at all :)

Reply:
Ah, OK. No problem. :)